public/app/features/dashboard/panel_editor/QueryOptions.tsx

```diff
@@ -300,7 +300,7 @@ export class QueryOptions extends PureComponent<Props, State> {
       >
         {this.renderMaxDataPointsOption()}
         {this.renderIntervalOption()}
-        {this.renderCacheTimeoutOption()}
+        {this.props.datasource.meta.queryOptions.cacheTimeout && this.renderCacheTimeoutOption()}
```

Comment:
think queryOptions is optional, this should cause increase in strictNull errors,  is there we can move this condition into renderCacheTimeoutOption ?

Reply:
Oh, yes can have a look